README.md

```diff
@@ -15,3 +15,17 @@ To generate and view documentation, follow the following steps:
   - `make html`
 - Point your browser to the generated HTML page:
   - `firefox _build/html/index.html`
+
+To create a new test, write a C program with a file name ending in "test.c"
+in the `test` directory.
+
+To run unthreaded tests, execute the following. Note that -U is required to undefine
```

Comment:
There are certain tests that we'll want to have for both the multi-threaded and single-threaded runtime. Would we just have to replicate these tests? Or might there be a more elegant solution for this?

Reply:
Currently, the tests under "general" are executed always, regardless of whether the runtime is single-threaded or multi-threaded. Does that suffice?

If tests become too expensive to run, then we could make an additional category for tests that can be run separately.

Reply:
Not sure... There are differing implementations (reactor.c vs reactor_threaded.c) of various functions that should functionally behave the same and thus would require verbatim copies of certain tests, with the difference only being the imports. This seems like a situation where the Cmake potentially could help.

Reply:
I think that use case should be covered if those tests are placed in "general." In that case, you could run `cmake .. -UNUMBER_OF_WORKERS` to run the tests in "general", compiling and linking `.c` files from the single-threaded runtime, and you could run `cmake .. -DNUMBER_OF_WORKERS=2` (or something else nonzero) if you want to run the same tests, compiling and linking `.c` files from the multithreaded runtime.

It works on my machine, at least.

Reply:
OK, yes, that sounds good. Thanks!